frontend/src/services/authService.js

```
@@ -31,28 +39,38 @@ const AuthService = {
-      return response.data;
+      const refresh_token = localStorage.getItem(REFRESH_KEY);
+      if (!refresh_token) {
+        throw new Error('No refresh token available');
+      }
+      const response = await api.post('/auth/refresh', { refresh_token });
+      const { access_token, refresh_token: new_refresh } = response.data;
+      localStorage.setItem(TOKEN_KEY, access_token);
+      localStorage.setItem(REFRESH_KEY, new_refresh);
+      return { access_token, refresh_token: new_refresh };
     } catch (error) {
       throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)